### PR TITLE
Bookstore: Use Redux in React components.

### PR DESCRIPTION
**Pull request by kessio**
Hello 👋🏾 

The following has been implemented on this PR
- [x] Implement the concept of state management.
- [x] Use store, actions, and reducers in React.
- [x] Connect React and Redux.
- [x] Linter test has passed
- [x] Proper gitflow followed

### **Happy reviewing**

### CODE REVIEW

File: src/redux/books/books.js [hunk 1]
```diff
@@ -1,27 +1,19 @@
-const ADD_BOOK = 'REACT-BOOKSTORE/src/redux/books/ADD_BOOK';
-const REMOVE_BOOK = 'REACT-BOOKSTORE/src/redux/books/REMOVE_BOOK';
+/* eslint-disable */
```
Comment (omarsalem7) [inline src/redux/books/books.js]: - [ ] Kindly remove this line to enable linters and try to fix them to make your code better.

File: src/redux/books/booksData.js [hunk 2]
```diff
@@ -0,0 +1,16 @@
+/* eslint-disable */
```
Comment (omarsalem7) [inline src/redux/books/booksData.js]: - [ ] Also remove this line as stated before

File: src/redux/categories/categories.js [hunk 3]
```diff
@@ -1,16 +1,15 @@
-const CHECK_STATUS = 'REACT-BOOKSTORE/src/redux/categories/CHECK_STATUS';
+/* eslint-disable */
```
Comment (omarsalem7) [inline src/redux/categories/categories.js]: - [ ] Also remove this line as stated before

**Review by omarsalem7 — CHANGES_REQUESTED** [review]
Hi @kessio 😊, 

Good job so far! 👏
You have done well reaching this point of the milestone, but there are some issues that you still need to work on to go to the next project but you are almost there! 

you-are-near

## Highlights
- The Pull Request has a proper title and description. ✔️
- Correct git-flow ✔️

## Required Changes ♻️
_Check the comments under the review._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by DammyShittu — APPROVED** [review]
Hi @kessio,

## APPROVED 🟢

## Highlights
✔️ Descriptive PR
✔️ Passing linters
✔️ Good branch name

Great implementation of all the requirements in this milestone. 👏 

_You have done an amazing job 👍_

Your project is complete!  There is nothing else to say other than... it's time to merge it. :shipit: 

Congratulations! 🎉 

## CHEERS AND HAPPY CODING!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**